### DISCUSSION TITLE
Where is Alex? Is this project still alive?

### DISCUSSION THREAD

**Discussion by StarMonk** [uNetworking/uWebSockets.js#842] [General]
As the title says, I see all of Alex's messages as "Ghost" - I am not sure if he is still around to maintain this project.

Any idea?

**Comment by uNetworkingAB** [comment]
I try not to live my life on social media so every now and then I scrub my data. I deleted that user since it had data going back to 2009, which is pretty insane to let big tech have that much on you.
